vhd/packer/sysprep.ps1

```diff
@@ -27,10 +27,10 @@ $values | ForEach-Object {
 }
 
 # run Sysprep
-if( Test-Path $Env:SystemRoot\\system32\\Sysprep\\unattend.xml ) {  Remove-Item $Env:SystemRoot\\system32\\Sysprep\\unattend.xml -Force }
-& $env:SystemRoot\\System32\\Sysprep\\Sysprep.exe /oobe /generalize /mode:vm /quiet /quit
+#if( Test-Path $Env:SystemRoot\\system32\\Sysprep\\unattend.xml ) {  Remove-Item $Env:SystemRoot\\system32\\Sysprep\\unattend.xml -Force }
```

Comment:
why comment this line? do we want to keep `unattended.xml`?

Reply:
This will not be needed, Image Builder service will cover this part